### PR TITLE
Change the way hostname maybe reset after installation

### PR DESCRIPTION
**Pull request by asmorodskyi**
Change the way hostname maybe reset after installation

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@DimStar77 , @Vogtinator , @okurz , @pevik  - WDYT about this version ?

File: variables.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ REPO_* | string | | Url pointing to the mirrored repo. REPO_0 contains installat
 RESCUECD | boolean | false | Indicates rescue image to be used.
 RESCUESYSTEM | boolean | false | Indicates rescue system under test.
 ROOTONLY | boolean | false | Request installation to create only the root account, no user account.
+RESET_HOSTNAME| boolean | false | If set to true content of /etc/hostname file will be ereased
```
Comment (Vogtinator) [inline variables.md]: Typo: "ereased"

**Comment by Vogtinator** [comment]
If this works, it's indeed much better, yes

File: variables.md (hunk 1 above)
Comment (asmorodskyi) [reply variables.md]: fixed , thanks

**Comment by asmorodskyi** [comment]
> Acked. Out of curiosity, why do you need it anyway? In which test is it used?

In a fact every multimachine test suppose to do this , or at least to do something with a fact that all tests in LAN has hostname=susetest. Once I was debuging scenario where DHCP server with hostname=susetest was requested by two machines with with hostname=susetest to give them hostname , Which not a nice thing ...

**Comment by pevik** [comment]
@asmorodskyi good point :). So what happen when no hostname is in `/etc/hostname`? Taken some from `/var/lib/wicked/lease-*-dhcp-ipv4.xml`? And what about with hostname record in `/etc/hosts`? I wonder if there is some smart solution for multimachine testing.

**Pull request merged**